Lib/test/support/__init__.py

```diff
@@ -1491,6 +1491,10 @@ def get_socket_conn_refused_errs():
         # bpo-31910: socket.create_connection() fails randomly
         # with EADDRNOTAVAIL on Travis CI
         errors.append(errno.EADDRNOTAVAIL)
+    if hasattr(errno, 'EHOSTUNREACH'):
+        # bpo-37583: socket.create_connection() fails always
+        # with EHOSTUNREACH on Virtual Machine
```

Comment:
This comment is wrong. The issue is specific to your VM config and it's unrelated to VMs in general. I suggest this comment instead:

```
# bpo-37583: The destination host cannot be reached
```

In the issue you wrote "Looks it's a problem of security group." maybe you can investigate to write a more accurate comment, but it's not required ;-)

Reply:
Thanks, victor. Your comment is good enough:)